packages/core/src/util/gestureUtils.ts

```
@@ -66,27 +66,27 @@ import type Cell from '../view/cell/Cell';
  * should be scaled according to the graph scale. If this is true, then
  * the offsets will also be scaled. Default is false.
  * @param highlightDropTargets Optional boolean that specifies if dropTargets
  * should be highlighted. Default is true.
  * @param getDropTarget Optional function to return the drop target for a given
- * location (x, y). Default is {@link Graph.getCellAt}.
+ * location (x, y). Default is {@link AbstractGraph.getCellAt}.
  */
 export const makeDraggable = (
   element: Element,
-  graphF: Graph | Function,
+  graphF: AbstractGraph | Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)